docs/deploy.md

```diff
@@ -140,24 +135,38 @@ This project is in it's early stages of development and hence there are features
 
       1. Update the `gcp_project_id` portion to your GCP Project Id.
       1. Change `contents_of_service_account_json` to be the entire contents of your GCP Service Account JSON.
-</br>
 
-1. Run the install script with your "CF Install Values" file.
+1. Run the following commands to install Cloud Foundry on your Kubernetes cluster.
+
+      i. Render the final K8s template to raw K8s configuration
+      ```console
+      ytt -f config -f /tmp/cf-values.yml > /tmp/cf-for-k8s-rendered.yml
+      ```
+      > cf-for-k8s uses [ytt](https://github.com/k14s/ytt) to create and maintain reusable YAML templates. You can visit the ytt [playground](https://get-ytt.io/) to learn more about it's templating features. 
+      > In the above command, `ytt` can take a folder e.g. `config` or file via `-f`. See all options by running `ytt help`.
+    
+      ii. Install using `kapp` and pass the above K8s configuration file
+      ```console
+      kapp deploy -a cf -f /tmp/cf-for-k8s-rendered.yml -y
+      ```
+      > cf-for-k8s uses [kapp](https://github.com/k14s/kapp) to manage it's lifecycle. `kapp` will first show you a list of resources it plans to install on the cluster and then will attempt to install those resources. `kapp` will not exit untill all resources are installed and their status is running. See all options by running `kapp help`.
 
+   Once you run the command, it should take about ~10-15mins dependning on your cluster bandwidth, size. `kapp` will provide updates on pending resource creations in the cluster (pods, lbs) and will wait till all resources are created and running. Following shows a sample output from `kapp`. 
```

Comment:
I'd change this to:

Once you run the command, it should take about 10 minutes depending on your cluster bandwidth, size. `kapp` will provide updates on pending resource creations in the cluster and will wait until all resources are created and running. Here is a sample snippet from `kapp` output: